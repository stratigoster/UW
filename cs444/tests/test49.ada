-- no errors here

PACKAGE main is
  i: integer;
END main;

PACKAGE body main IS
  x : integer;
  procedure bar (x:integer;y:float) is
  begin
    i := x;
    return 5;
--  ^^^^^^^^
-- procedures don't have return values
  end bar;
begin
  if true then
    begin
      bar(i,3.0); -- yay procedure calls work
      x := 5;
    end;
  else
    x := 6;
  end if;
END main;
