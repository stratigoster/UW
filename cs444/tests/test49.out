***ERROR***:12:14:	Not in a function to be able to return from

------
Compilation error
------
